doc/instances.md

```diff
@@ -54,7 +54,7 @@ limits.kernel.\*                            | string    | -                 | no
 limits.memory                               | string    | - (all)           | yes           | -                         | Percentage of the host's memory or fixed value in bytes (various suffixes supported, see below)
 limits.memory.enforce                       | string    | hard              | yes           | container                 | If hard, instance can't exceed its memory limit. If soft, the instance can exceed its memory limit when extra host memory is available
 limits.memory.hugepages                     | boolean   | false             | no            | virtual-machine           | Controls whether to back the instance using hugepages rather than regular system memory
-limits.memory.swap                          | boolean   | true              | yes           | container                 | Whether to allow some of the instance's memory to be swapped out to disk
+limits.memory.swap                          | boolean   | true              | yes           | container                 | Controlers whether to encourage/discourage swapping less used pages for this instance
```

Comment:
@stgraber whoops was a bit quick off the mark there, there's a typo in "Controlers".